lib/form/paragraph3.flow

```diff
@@ -893,3 +915,20 @@ setWordText(w: ParaWord, t: string) -> ParaWord {
 setWordForm(word: ParaWord, form: GhostForm) -> ParaWord {
 	ParaWord(word.word, word.metrics, word.ghosted, form, word.interactivityIdM);
 };
+
+makeWigiStoryWidthManager(paragraphs : int, tightWidth : bool) -> WigiStoryWidthManager {
+	paragraphWidths = generate(0, paragraphs, \__ -> make(0.0));
+	if (tightWidth) {
+		storyWidth = arrayMaxu(paragraphWidths);
```

Comment:
Looks like you missed to unsubscriber of arrayMaxu, or I missunderstand something.

Reply:
I didn't call unsubsriber, it is true. There is no good and easy to use place to call it.
We create WigiStoryWidthManager in the doWigifyStory but the real form is created later and we can't call disposer in the usual place like form's Constructor's destructor because it would be called when the form is hided and then after shows the subscription will be lost.

but I think that this will not introduce memory leak because GC will collect all this elements at at once when the WigiStoryWidthManager is collected.
maybe I'm wrong. @yegordovganich , what do you think, will it produce memory leak or not?

Reply:
I don't think it's good idea to leave it w/o unsubscriber call. The unsubscriber is not necessary to be called by constructor, but manually.

Reply:
@alstrup , I need your advice about if the unsubscribe call really required in this case.
If I'm not mistaken, it is redundant here because subscription happens inside function makeWigiStoryWidthManager and arrayMaxu is used on behaviors created there too, so when the object WigiStoryWidthManager will be available for GC it collects it along with subscription. The WigiStoryWidthManager will be ready for GC when the resulted MWigify material will be removed from the screen.

Reply:
fixed